docs/WindowsBuild.md

```diff
@@ -0,0 +1,176 @@
+# Building Swift on Windows
+
+This document describes how to build Swift for Windows natively. See [the
+Windows doc](./Windows.md) for more about what is possible with Swift on
+Windows.
+
+There are two supported ways to build Swift on Windows, they are
+
+1. Using [`clang-cl`](https://clang.llvm.org/docs/UsersManual.html#clang-cl)
+1. Using the Microsoft Visual C++ compiler (MSVC)
+
+`clang-cl` is recommended over MSVC for building Swift on Windows.
+Although it is possible to build the compiler and the standard library with
+MSVC, and use those built products to compile a Swift program, it won't be
+possible to run the binary without seperately obtaining the Swift runtime. On
+the other hand, `clang-cl` is able to build the runtime, which makes it
+possible to build and run all the components required for Swift natively on
+Windows.
+
+## MSVC
+- Windows doesn't currently have a build script. You'll need to run commands
+  manually to build Swift on Windows.
+- Windows support for Swift is a work in progress and may not work on your
+  system, but it has been tested.
+- Using the latest Visual Studio version is recommended. Swift may fail to
+  build with older C++ compilers.
+
+### 1. Install dependencies
+- Make sure to add Python, CMake and Ninja to your `Path` environment variable
+1. Latest version (2.7.12 tested) of [Python
+   2](https://www.python.org/downloads/)
+2. Latest version (3.7.0-rc3 tested) of [CMake](https://cmake.org/download/)
+3. Latest version (1.7.1 tested) of
+   [Ninja](https://github.com/ninja-build/ninja/releases/latest)
+4. Latest version (2015 Update 3 tested) of [Visual
+   Studio](https://www.visualstudio.com/downloads/)
+
+- Make sure to include `Programming Languages|Visual C++` and `Windows and Web
+  Development|Universal Windows App Development|Windows SDK` in your
+  installation.
+
+### 2. Clone the repositories
+1. Create a folder to contain all the Swift repositories
+2. `apple/swift-cmark` into a folder named `cmark`
```

Comment:
Since item 1 has a verb ("Create a folder..."), the remainder of the list needs verbs ("_Clone_ `apple/swift-cmark`...").